com.adamcake.Bolt.yml

```diff
@@ -0,0 +1,62 @@
+app-id: com.adamcake.Bolt
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: bolt
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --env=PATH=/app/bin:/usr/bin:/app/jre/bin
+modules:
+  - name: python-pillow
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links=file://${PWD} --prefix=${FLATPAK_DEST} pillow --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/0f/8b/2ebaf9adcf4260c00f842154865f8730cf745906aa5dd499141fb6063e26/Pillow-10.0.0.tar.gz
+        sha256: 9c82b5b3e043c7af0d95792d0d20ccf68f61a1fec6b3530e718b688422727396
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+  - shared-modules/gtk2/gtk2.json
+  - name: openssl
+    buildsystem: simple
+    build-commands:
+      - ./config --prefix=/app --openssldir=/app/ssl shared zlib
+      - make -j $FLATPAK_BUILDER_N_JOBS
+      - make install
+    sources:
+      - type: git
+        url: https://github.com/openssl/openssl.git
+        tag: OpenSSL_1_1_1v
+        commit: 9398e4038c27d5ef15111c61e984d4d2be8a52a8
+  - name: bolt
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBOLT_BINDIR=bin
+    post-install:
+      - install -Dm644 com.adamcake.Bolt.appdata.xml /app/share/appdata/
```

Comment:
```suggestion
      - install -Dm644 com.adamcake.Bolt.appdata.xml -t /app/share/appdata/
```

Reply:
also the file can't be found because it's not availabe in git at the tag. Same for the .desktop.